openwisp_monitoring/monitoring/tests/test_admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from . import TestMonitoringMixin
+
+
+class TestAdmin(TestMonitoringMixin, TestCase):
+
+    def _login_admin(self):
```

Comment:
@PabloCastellano, there's an additional blank line here, which led to all corresponding builds on the `dev` branch to fail due to `qa-checks` failure. No worries fixed it in #150 :smile:.